test/e2e/localnet-underlay.go

```diff
@@ -45,6 +45,20 @@ func setupUnderlay(ovsPods []v1.Pod, portName string, nadConfig networkAttachmen
 	return nil
 }
 
+func reconfigureVLANAccessPort(ovsPods []v1.Pod, portName string, oldVLANID, newVLANID int) error {
```

Comment:
The commit change is wrong is reapeating the exclude-subnets one.

Reply:
ah yes thank you